src/components/checkbox/index.tsx

```diff
@@ -1,12 +1,13 @@
 import { FC } from 'react'
 
-import { CheckBoxInput, CheckBoxLabel, CheckBoxWrapper } from './styles'
+import { CheckBoxInput, CheckBoxLabel, Checkmark } from './styles'
 
 type Props = Omit<React.InputHTMLAttributes<HTMLInputElement>, 'type'> & { labelTitle: string }
```

Comment:
I can't use parametr `title` in `label` and  `input` because title is a toast when user handle mouse over, and because I changed `labelTitle` to `label`, (`label` doesn`t exist in type `InputHTMLAttributes`)